src/core_plugins/kibana/public/management/sections/indices/_edit.html

```diff
@@ -32,22 +32,35 @@
       </div>
 
       <form role="form">
-        <input aria-label="Filter" ng-model="fieldFilter" class="form-control span12" type="text" placeholder="Filter" />
+        <input ng-disabled="state.tab == 'sourceFilters'" aria-label="Filter" ng-model="fieldFilter" class="form-control span12" type="text" placeholder="Filter" />
       </form>
 
       <br />
 
+      <!-- tab list -->
       <ul class="nav nav-tabs">
         <li class="kbn-management-tab" ng-class="{ active: state.tab === fieldType.index }" ng-repeat="fieldType in fieldTypes">
           <a ng-click="changeTab(fieldType)">
             {{ fieldType.title }}
-            <small>({{ fieldType.count }})</small>
+            <small ng-if="fieldType.count">({{ fieldType.count }})</small>
           </a>
         </li>
       </ul>
 
-      <indexed-fields ng-show="state.tab == 'indexedFields'" class="fields indexed-fields"></indexed-fields>
-      <scripted-fields ng-show="state.tab == 'scriptedFields'" class="fields scripted-fields"></scripted-fields>
+      <!-- tabs -->
+      <indexed-fields
+        ng-show="state.tab == 'indexedFields'"
+        class="fields indexed-fields">
+      </indexed-fields>
+      <scripted-fields
+        ng-show="state.tab == 'scriptedFields'"
+        class="fields scripted-fields">
+      </scripted-fields>
+      <field-filters
+        ng-show="state.tab == 'sourceFilters'"
+        index-pattern="indexPattern"
+        class="fields field-filters">
+      </field-filters>
```

Comment:
You can place the closing bracket and the closing tags on the same line, as described here https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md